### ISSUE TITLE
Conflicting TXT records when using a certificate both as wildcard and for the naked domain

### ISSUE THREAD

**Issue by leonardder** [certbot/certbot#5772]
## I ran the following command:
`certbot certonly --manual -d example.com -d *.example.com --server https://acme-v02.api.letsencrypt.org/directory --text --preferred-challenges dns-01`

## This produced the following conflicting output regarding txt records
```
-------------------------------------------------------------------------------
Please deploy a DNS TXT record under the name
_acme-challenge.example.com with the following value:

Gv_T2cieXWT8XlwyrCFc4QP71tYpYqWktjhK8yuYibY

Before continuing, verify the record is deployed.
-------------------------------------------------------------------------------
Press Enter to Continue

-------------------------------------------------------------------------------
Please deploy a DNS TXT record under the name
_acme-challenge.example.com with the following value:

il_yIUyjjd8A0GBR08bPwOOQylV39xLN-PE_b187fAo

Before continuing, verify the record is deployed.
-------------------------------------------------------------------------------
Press Enter to Continue
```

To summarize, the client seems to ask me to create two txt records with the same name, but different contents. This is Certbot 0.22

**Comment by sydneyli** [comment]
That's correct!  You need to set two separate TXT records  for `_acme-challenge.example.com`, each with a different value. We're working on improving our documentation about this here: #5729. If you're not sure how to do this, I would recommend asking the folks over at https://community.letsencrypt.org-- there's a larger audience of people there who will be able to help out.

**Comment by dj0x** [comment]
I solved, I discover that have to be created just one record TXT (_acme-challenge.*) with two values (hash given by certbot) separated by line. After run certbot, remember to restart your webserver.

I think depends on your DNS server to setup this. I did it in Route 53 - AWS and this fix this problem.

Obs: Considering wait some seconds (+30) after change your records.

The problem has fixed here:

https://stackoverflow.com/questions/54654135/letsencrypt-certbot-rejects-dns-txt-record-for-wildcard-certificate/55658516#55658516
